_posts/en/newsletters/2021-04-07-newsletter.md

```diff
@@ -79,7 +79,10 @@ BOLTs][bolts repo].*
   of code. Eclair supported Electrum for mobile wallets, however a mobile wallet
   optimized LN implementation, [Eclair-kmp][eclair-kmp github], is now recommended.
 
-- [Eclair #1751][] Add blocking option to payinvoice API FIXME:dongcarl
+- [Eclair #1751][] adds a `blocking` option to the `payinvoice` command which
+  causes calls to `payinvoice` to block until the payment is completed.
```

Comment:
Wouldn't it just be the specific call encumbered with the `blocking` option that gets blocked?

Reply:
Oh, I see, the subsequent calls also get held up until the one with the `blocking` option is resolved?